tests/interval_log.rs

```diff
@@ -250,10 +268,11 @@ mod tests {
             }
         }
 
+        println!("{}", ::std::str::from_utf8(&log_buf).unwrap());
```

Comment:
I assume this is just leftover from debugging?

Reply:
Doh!